code/modules/projectiles/ammunition/_ammunition.dm

```diff
@@ -67,17 +67,32 @@
 /obj/item/ammo_casing/proc/add_notes_ammo()
 	// Try to get a projectile to derive stats from
 	var/obj/projectile/exam_proj = projectile_type
+	// projectile damage multiplier for guns with snowflaked damage multipliers
+	var/proj_damage_mult = 1
 	if(!ispath(exam_proj) || pellets == 0)
 		return
 
+	// are we in an ammo box?
+	if(isammobox(loc))
+		var/obj/item/ammo_box/our_box = loc
+		// is our ammo box in a gun?
+		if(isgun(our_box.loc))
+			var/obj/item/gun/our_gun = our_box.loc
+			// grab the damage multiplier
+			proj_damage_mult = our_gun.projectile_damage_multiplier
+	// if not, are we just in a gun e.g. chambered
+	else if(isgun(loc))
+		var/obj/item/gun/our_gun = loc
+		// grab the damage multiplier.
+		proj_damage_mult = our_gun.projectile_damage_multiplier
 	var/list/readout = list()
+	if(!proj_damage_mult || proj_damage_mult <= 0)
```

Comment:
```suggestion
	if(proj_damage_mult <= 0 || (initial(exam_proj.damage) <= 0 && initial(exam_proj.damage) <= 0))
```
Also, it's worth mentioning you could/should store these initial() values in a variable if they're checked for several times in the proc.